scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<p>Nit: a small training <strong>dataset</strong></p><p> </p><p>Do we need to have the last two sentences on download/unzip, and setting the path?  These steps are pretty self-explanatory in the code. </p><p>Also the sentence on path is a little confusing, as it could make the reader think of the PATH environment variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>